### ISSUE TITLE
[Topic] Tables in theme and plugin development

### ISSUE THREAD

**Issue by rianrietveld** [wpaccessibility/wp-a11y-docs#144]
URL: https://wpaccessibility.org/docs/topics/code/tables/
Code: https://github.com/wpaccessibility/wp-a11y-docs/blob/main/docs/topics/code/tables/index.md

Please read [How to contribute](https://wpaccessibility.org/docs/contribute/) before you start.

## Tasks

Rename the page to: Tables in frontend development

This content needs to be reviewed and restructured.
Keep as much of the current text as you can.
Check the links.

Also: this is discussed in Wp code and Gutenbger, make sute we have no contradicting or incomplete content. The content should help peeople who work om:
https://github.com/WordPress/gutenberg/issues/32394
https://github.com/WordPress/gutenberg/issues/6923
https://github.com/WordPress/gutenberg/issues/57278
https://github.com/WordPress/gutenberg/issues/24205

Bohdan Smaha wrote a [Table V2 Accessibility Requirements](https://hackmd.io/@captain-bohdan/BkZUSOkvfe) on HackMD.

Related discussion in SLack
https://wordpress.slack.com/archives/C02RP4X03/p1786898895230959

See for the structure of an accessibility topic: 
Write documentation: [Content structure of an accessibility topic](https://wpaccessibility.org/docs/contribute/write/#content-structure-of-an-accessibility-topic)
